### ISSUE TITLE
multi gpu inference with emo_asr.models.ASRConvCTCModel

### ISSUE THREAD

**Issue by paklau99988** [NVIDIA/NeMo#966]
Hello, 
For inference I am using:
``
 nf = nemo.core.NeuralModuleFactory()

    # Instantiate the model which we'll train    
    asr_model = nemo_asr.models.ASRConvCTCModel.from_pretrained(model_info="QuartzNet15x5-Zh")
    asr_model.eval()

    eval_data_layer = nemo_asr.AudioToTextDataLayer(
#         manifest_filepath="test_single.json",
        manifest_filepath=corpus,        
#         manifest_filepath="./ocean-to-aishell.json",        
        labels=asr_model.vocabulary,
        batch_size=16,
        trim_silence=True,
        shuffle=True,
        normalize_transcripts=False,
    )
``

How can I change the number of GPU's used for inference? CUDA_VISIBLE_DEVICES did not change anything. 
Also how do I change the number of the PyTorch dataloader workers during inference? 

Thanks,
